search-app/docs/api.md

```diff
@@ -5410,9 +5410,9 @@ Example un-healthy response body:
 
 ### <a name="associate-any-concepts"></a> Associate any concepts
 
-A new association API has been developed to achieve the goal of being able to associate a concept of any type, with or without revision, to one or more other concepts of any type, with or without revisions. The new association API also allows the associations to include an optional data payload, whose purpose is to describe the association itself. Associations which do not initially have an association data payload, may have it added through an association update. A concept can only be associated with another concept either with or without revisions, not both. A concept can not be associated to itself, even with different revisions. It's worth noting that associations between collections and services/tools/variables can not be made through the new association API because these associations require different business rules and there exist complexites with the way how certain fields of the association entries are constructed. Until we resolve these issues, these existing associations will continue to be made through the existing association API.
- 
-#### <a name="concept-associations"></a> Concept associations 
+A new association API has been developed to achieve the goal of being able to associate a concept of any type, with or without revision, to one or more other concepts of any type, with or without revisions. The new association API also allows the associations to include an optional data payload, whose purpose is to describe the association itself. Associations which do not initially have an association data payload, may have it added through an association update. A concept can only be associated with another concept either with or without revisions, not both. A concept can not be associated to itself, even with different revisions. It's worth noting that associations between collections and services/tools/variables cannot be made through the new association API because these associations require different business rules. These existing associations will continue to be made through the existing association API.
```

Comment:
is there any way we can split this paragraph up? It's information dense, and throws a lot at the user all at once.

Reply:
That is a very good point I think I could split it into two section: The top level information about the generic association API and then another section with something along the lines of "important-notes" that breakdown some of the subtleties of what can be done,